### PR TITLE
fix(popup): add namespace to open and close events (VIV-1198)

### PR DESCRIPTION
**Pull request by rinaok**

### CODE REVIEW

File: libs/components/src/lib/menu/menu.spec.ts [hunk 1]
```diff
@@ -235,7 +235,7 @@ describe('vwc-menu', () => {
 	describe('events', () => {
 		it('should fire an event on popup open', async () => {
```
Comment (YonatanKra) [inline libs/components/src/lib/menu/menu.spec.ts:236]: I actually don't understand this test.
Does it test the `popup`? What does the menu need to do in the case of `open`?

File: libs/components/src/lib/menu/menu.spec.ts [hunk 2]
```diff
@@ -245,7 +245,7 @@ describe('vwc-menu', () => {
 		it('should fire an event on popup close', async () => {
```
Comment (YonatanKra) [inline libs/components/src/lib/menu/menu.spec.ts:245]: Again, it seems like this test tests the `popup` and not the `menu`.
What does the menu need to do in the case of `popup:close`?

File: libs/components/src/lib/popup/popup.spec.ts [hunk 3]
```diff
@@ -148,7 +148,7 @@ describe('vwc-popup', () => {
 
 		it('should fire open & close events', async function () {
```
Comment (YonatanKra) [inline libs/components/src/lib/popup/popup.spec.ts:149]: This tests only `open`. 
```suggestion
		it('should fire vwc-popup:open event', async function () {
```

File: libs/components/src/lib/popup/popup.spec.ts [hunk 4]
```diff
@@ -169,7 +169,7 @@ describe('vwc-popup', () => {
 		it('should fire close event', async function () {
```
Comment (YonatanKra) [inline libs/components/src/lib/popup/popup.spec.ts:169]:
```suggestion
		it('should fire "vwc-popup:close" event', async function () {
```

**Review by YonatanKra — CHANGES_REQUESTED** [review]
Really small changes.
Thanks!

File: libs/components/src/lib/menu/menu.spec.ts (hunk 1 above)
Comment (rinaok) [reply libs/components/src/lib/menu/menu.spec.ts:236]: Updated the test

File: libs/components/src/lib/menu/menu.spec.ts (hunk 2 above)
Comment (rinaok) [reply libs/components/src/lib/menu/menu.spec.ts:246]: Updated the test

**Pull request merged**